### DISCUSSION TITLE
Coolify intergation

### DISCUSSION THREAD

**Discussion by compgeniuses** [Infisical/infisical#1324] [Feature Request]
Would you guys be open to implementing a coolify intergation for this secret manager, would be wonderful

coolify can be found here: https://coolify.io/

i think maybe a kind sponsor gesture to them might get them to build a service and help you work together on the integration mayb

**Comment by akhilmhdh** [comment]
hey @compgeniuses kindly raise an issue for this. Integrations are to an extend community driven. So probably people will discover this

**Reply by rennokki** [reply]
@compgeniuses I have added Infisical on Coolify 1 month ago or so.

**Comment by compgeniuses** [comment]
@rennokki as a service, or with possibility of managing coolify secrets i infisical.

**Reply by rennokki** [reply]
You can deploy your Infisical instance on Coolify using one-click installs. Or you refer to having Docker containers running on Coolify and have an Infisical agent that injects secrets into other one-click services??

**Reply by compgeniuses** [reply]
i meant, USing infisical to manage coolify continers secrets. that way i have a centralised Place for my secrets

**Reply by mfyz** [reply]
Seconding @comp. Having native integration of infisical i. Coolify to manage all secrets in deployment pipelines and in runtime would be killer.

**Comment by RatulMaharaj** [comment]
I really would love for coolify to be a 3rd Party target for infisical `Secret Syncs` similar to what exists currently for vercel, cloudflare workers etc

<img width="676" height="723" alt="image" src="https://github.com/user-attachments/assets/70a24e55-9a73-41b9-9a41-389a3c1476c0" />

**Reply by RatulMaharaj** [reply]
For anyone interested, I worked around this by embedding the infisical cli binary into my containers and then dynamically pulling the vars into the container at runtime with a custom entrypoint script (using infisicals machine identity auth) . 

That way you don't actually need a sync, you can just manage your secrets in infisical.

**Reply by wertania** [reply]
I do it the same way.

**Comment by julianbrennum** [comment]
This would be great, i was just looking if this was possible.

**Comment by wcatbb** [comment]
Also looking for this. It's a natural fit and would be a huge win.
